pixart.js

```diff
@@ -1 +1,18 @@
+$('#set-color').on('click', function(evt) {
+  evt.preventDefault();
+  $('.brush').css('background', $('.input').val());
+})
 
+$('#set-color').on('keypress', function(evt) {
+  evt.preventDefault();
+  $('.brush').css('background', $('.input').val());
+})
+
```

Comment:
`.on('submit', function(evt){})`
listens for both enter and click events on inputs.